src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,36 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import users from '../../api/users.json';
+import comments from '../../api/comments.json';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => {
+  const postByUser = users.find(user => post.userId === user.id);
+  const postCommetns = comments.filter(comment => post.id === comment.postId);
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={postByUser} />
```

Comment:
Use is possibly indefined, add check if use exist
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find
![2024-02-15_15h55_14](https://github.com/mate-academy/react_static-list-of-posts-js/assets/61904995/3ef9c315-a6b8-4f72-b3b0-83c6b60b54b9)